coderd/clissh.go

```diff
@@ -0,0 +1,18 @@
+package coderd
+
+import (
+	"net/http"
+
+	"github.com/coder/coder/coderd/httpapi"
+)
+
+// @Summary CLI SSH Config
+// @ID cli-ssh-config
+// @Security CoderSessionToken
+// @Produce json
+// @Tags General
+// @Success 200 {object} codersdk.SSHConfigResponse
+// @Router /deployment/ssh [get]
+func (a *API) cliSSHConfig(rw http.ResponseWriter, r *http.Request) {
```

Comment:
nit: Let's put this in `deployment.go`

Reply:
Can I just move this in there too?

https://github.com/coder/coder/blob/a1dd7d491367626de52d65235aba3e33a0d7ba68/coderd/buildinfo.go#L10-L23

Reply:
def